src/NServiceBus.Transport.RabbitMQ.Tests/TestApprover.cs

```diff
@@ -0,0 +1,33 @@
+﻿#if NET452
+namespace NServiceBus.Transport.RabbitMQ.Tests
+{
+    using System.IO;
+    using ApprovalTests;
+    using ApprovalTests.Namers;
+
+    static class TestApprover
```

Comment:
Why has this class been introduced? Is it purely to move the approved API to a subfolder?

Reply:
Discussed offline: this is so that we can copy the build artifacts to a dependent build on another agent during CI. The default behaviour is walk up the call stack looking for source file info. This swaps that out for using a known file location relative to the assembly.